exersices/Ex05/bounded_sim.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Sindre Tallaksrud'
+__email__ = 'sindrtal@nmbu.no'
+
+
+from walker_sim import Walker, Simulation
+
+
+class BoundedWalker(Walker):
+
+    def __init__(self, start, home, left_limit, right_limit):
+
+        """
+        Initialise the walker
+        Arguments
+        ---------
+        start : int
+            The walker's initial position
+        home : int
+            The walk ends when the walker reaches home
+        left_limit : int
+            The left boundary of walker movement
+        right_limit : int
+            The right boundary  of walker movement
+        """
+        super().__init__(start, home)
+        self.init_pos = start
+        self.home = home
```

Comment:
You already stored these variables in `super().__init__(start, home)` so please remove these.